packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -367,6 +389,7 @@ export class DocumentDeltaConnection
 	}
 
 	protected disconnect(err: IAnyDriverError) {
+		clearInterval(this.trackLatencyTimer);
```

Comment:
you probably want to move it couple lines down, after next IF check

Reply:
We also probably do not want to call this API if trackLatencyTimer === undefined - I'm not sure if behavior it defined in the spec for this case (it probably works, but not sure if it's promised to us)